src/v/cluster/plugin_frontend.cc

```diff
@@ -476,6 +481,19 @@ errc plugin_frontend::validator::validate_mutation(const transform_cmd& cmd) {
                     loggable_string(out_name.ns()));
                   return errc::transform_invalid_create;
               }
+              if (
+                out_name.tp == model::kafka_audit_logging_topic
+                || out_name.tp == model::kafka_consumer_offsets_topic
+                || out_name.tp == model::schema_registry_internal_tp.topic) {
```

Comment:
This is good, but I do think you should also still reference the no produce topics list.  I apologize if my previous comment suggested otherwise, was not my intention.

Reply:
Thanks fixed - I'm not sure noproduce is actually used anywhere, and it might make sense to mark a topic as wasm writable only, but one should just use ACLs for that.